maven-projects/info/pom.xml

```diff
@@ -63,4 +63,22 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>${spotless-maven-plugin.version}</version>
+                <configuration>
```

Comment:
Is it possible to inherit this plugin from the parent instead of defining it explicitly?